src/wordpress-recommendations.lib.php

```diff
@@ -26,70 +25,237 @@
+        (filemtime(ABSPATH.'/wp-config.php') > strtotime('-12 months'))) {
+            unset($recommendations['wpSaltAgeDiscriminator']);
+        }
+
+        /*
+         * Check for standard administrator/admin account.
+         * @see https://sucuri.net/guides/wordpress-security/#uac
+         */
+        $usersWithAdminLogin = get_users(array(
+            'role' => 'administrator',
+            'login__in' => array('admin', 'administrator'),
+        ));
+        if (empty($usersWithAdminLogin)) {
+            unset($recommendations['adminBadUsername']);
+        }
+
+        /*
+         * Check if super-admin isn't being used for day-to-day operations.
+         * @see https://sucuri.net/guides/wordpress-security/#uac
          */
+        $wpUsersCount = count_users();
+        if ($wpUsersCount['total_users'] !== 1) {
+            unset($recommendations['lonelySuperAdmin']);
+        }
+
+        /*
+         * Check if 2FA plugin is being used.
+         * @see https://sucuri.net/guides/wordpress-security/#pwd
+         *
+         * NOTE: $wpPluginsInstalledName, $wpPluginsActivatedName, $wpPluginsDeactivatedName
+         * are created by this feature.
+         */
+        $wpPluginsInstalled = get_plugins();
+        $wpPluginsActivatedName = array();
+        $wpPluginsDeactivatedName = array();
+        foreach ($wpPluginsInstalled as $pluginPath => $pluginDetails) {
+            $wpPluginsInstalledName[] = $pluginDetails['Name'];
+            if (is_plugin_active($pluginPath)) {
+                $wpPluginsActivatedName[] = $pluginDetails['Name'];
+            } else {
+                $wpPluginsDeactivatedName[] = $pluginDetails['Name'];
+            }
+        }
+        // phpcs:disable Generic.Files.LineLength
+        $wp2faPluginsActive = preg_grep('/(2FA|2\sFactor|Two\sFactor|Two-Factor|2-Factor|Secure\sLogin|SecSign|Authentication|Wordfence\sSecurity|iThemes\sSecurity\sPro|Limit\sLogin|LDAP)/i', $wpPluginsActivatedName);
+        if (!empty($wp2faPluginsActive)) {
+            unset($recommendations['loginUnprotected']);
+        }
+        // phpcs:enable
 
-        // Check if php version needs to be upgraded.
-        if (version_compare(phpversion(), '7.1', '<')) {
-            $recommendations['PHPVersionCheck'] = array(
-                __('Upgrade PHP to a supported version', 'sucuri-scanner') =>
-                __('The PHP version you are using no longer receives security support and could be exposed to unpatched security vulnerabilities.', 'sucuri-scanner')
-            );
+        /*
+         * Check for unwanted extensions.
+         * @see https://sucuri.net/guides/wordpress-security/#apt
+         *
+         * NOTE: This check uses $wpPluginsDeactivatedName which is created by
+         * the 2FA plugin check.
+         */
+        // phpcs:disable Generic.Files.LineLength
+        if ((count(wp_get_themes()) < 2 || count($wpPluginsDeactivatedName) < 1) || is_multisite()) {
```

Comment:
This will show the forgottenExtension warning for every multisite installation. Maybe what you meant is:
```
if ((count(wp_get_themes()) < 2 || count($wpPluginsDeactivatedName) < 1) && !is_multisite())
```

Reply:
Oh, nevermind.